File: app/src/main/java/com/zhang/mydemo/java/ui/activity/JumpTextActivity.java
package com.zhang.mydemo.java.ui.activity;

import android.widget.TextView;

import com.zhang.mydemo.R;
import com.example.baselibiary.base.BaseActivity;
import com.zhang.mydemo.databinding.ActivityJumpTextBinding;

public class JumpTextActivity extends BaseActivity<ActivityJumpTextBinding> {

    @Override
    protected void initView() {
        findViewById(R.id.ivPageBack).setOnClickListener(v -> {
            killMyself();
        });
        TextView tvPageTitle = findViewById(R.id.tvPageTitle);
        tvPageTitle.setText("文字绘制轨迹");

    }

    @Override
    protected void initData() {

    }

    @Override
    protected void setListener() {

    }
}

File: app/src/main/java/com/zhang/mydemo/ui/activity/JumpTextActivity.kt
package com.zhang.mydemo.ui.activity

import android.os.Bundle
import com.zhang.mydemo.base.activity.BaseActivity
import com.zhang.mydemo.databinding.ActivityJumpTextBinding
import kotlinx.android.synthetic.main.layout_title.*

class JumpTextActivity : BaseActivity<ActivityJumpTextBinding>() {


    override fun initData() {}

    override fun setListener() {}

    override fun initView(savedInstanceState: Bundle?) {
        tvPageTitle.text = "文字绘制轨迹"

    }
}